src/buffer.c

```diff
@@ -4775,6 +4775,11 @@ build_stl_str_hl(
 	    get_rel_pos(wp, str, TMPLEN);
 	    break;
 
+	case STL_SHOWCMD:
+	    if (p_sc && STRCMP(opt_name, p_sloc) == 0)
```

Comment:
```c
case STL_SHOWCMD:
	    if (p_sc && STRCMP(opt_name, p_sloc) == 0)
		str = showcmd_buf;
	    break;
```

Not sure if this check is needed, I think `showcmd_buf` will always be empty here if those conditions are false anyways.